homeassistant/components/upb/__init__.py

```diff
@@ -68,6 +68,8 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
 class UpbEntity(Entity):
     """Base class for all UPB entities."""
 
+    _attr_has_entity_name = True
```

Comment:
upb seems to use `self._element.name` for both its device and entity?

That seems not correct in the new schema of things. If an entity should be the same as the device name, set the entity name to `None`.